cfme/tests/infrastructure/test_host_analysis.py

```diff
@@ -63,6 +63,8 @@ def host_with_credentials(provider, host_name):
 @pytest.mark.uncollectif(
     lambda provider, appliance:
     appliance.version == miq_version.UPSTREAM and provider.one_of(RHEVMProvider))
+@pytest.mark.meta(blockers=[BZ(1650179, forced_streams=['5.10'],
```

Comment:
Gotta import BZ, wasn't already imported in this module. Why travis is failing. :)

Reply:
Nice catch, thank you!